tests/unittests/test_rdsr_parsing.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+from pathlib import Path
+
+import numpy as np
+import pydicom
+
+from pyskindose.rdsr_normalizer import rdsr_normalizer
+from pyskindose.rdsr_parser import rdsr_parser
+
+phantom_path = Path(__file__).parent.parent.parent / "src" / "pyskindose" / "example_data" / "RDSR"
+
+
+def test_nr_irradiation_events_parsing_in_rdsr_parser():
+
+    # load SR file with known number of irradiation events
+    RDSR_file = pydicom.read_file(phantom_path / "s1.dcm")
+
+    expected = 24
+
+    # parse raw rdsr file
+    data_parsed = rdsr_parser(data_raw=RDSR_file)
+
+    test = len(data_parsed)
+
+    assert expected == test
+
+
+def test_xray_filter_material_parsing():
```

Comment:
Kind of nitpicky but you could be a bit more explicit in the name with what you're testing. What about the parsing is tested here? That any data is extracted. That both an aluminium and copper filter is extracted?